src/Illuminate/Support/NamespacedItemResolver.php

```diff
@@ -56,18 +56,12 @@ protected function parseBasicSegments(array $segments)
         // just pulling an entire group out of the array and not a single item.
         $group = $segments[0];
```

Comment:
You could also now also remove this line, and below do `return [null, $segments[0], $item];`?